### PR TITLE
Update Cyberia2019 to Cyberia2020

### PR DESCRIPTION
**Pull request by clement0010**

### CODE REVIEW

File: pages/cyberia.md [hunk 1]
```diff
@@ -1,36 +1,44 @@
...
+**Note**: The allocation of spaces are based on first-come-first-served basis based on the payment of camp fee. Your RSVP will only be confirmed upon completion of payment of camp fee. **No refund can be made upon payment.**
 
 Do look up for emails once you completed the RSVP and the payment!
 
-Sign up now at http://bit.ly/cyberia19 to upskill yourself! See you in the camp!
+Sign up now at [http://bit.ly/Cyberia_2020](http://bit.ly/Cyberia_2020) to upskill yourself! See you in the camp!
+
+<!-- Additional Information (?) -->
+<!-- # Section 2.1 - Additional Information & Sneak Peak & Main Event Page Redirect Link -->
+<!-- May include redirect link to event website and sneakpeak, or past photo -->
```
Comment (wfxronald) [inline pages/cyberia.md]: You will add this in a separate PR?

File: pages/cyberia.md [hunk 2]
```diff
@@ -40,12 +48,17 @@ Although we are a newly-formed company, our team is made up of veterans who have
...
 
 ## About the trainer: Ricky Soh
-A self-professed technophile, Ricky's fascination for computer software developed into a passion for sharing his knowledge in I.T. With more than 20 years in the I.T. training industry, Ricky's extensive experience comes from teaching as a polytechnic lecturer and corporate instructor.
 
-At Ngee Ann Polytechnic, Ricky taught programming languages as well as web design and web development. As a corporate instructor, Ricky has been an Adobe Certified Instructor and Adobe Certified Expert since 2006. Due to his numerous Adobe certifications, he has been conferred the highly coveted title of “Adobe Certified Instructor – Creative Suite Master”. Some of his major clients include GlaxoSmithKline, NOL, AIA, SPH, and Seagate. He has also collaborated with People's Association in organising and conducting I.T. courses for the public in community clubs.
+A self-professed technophile, Ricky’s fascination for computer software developed into a passion for sharing his knowledge in I.T. With more than 20 years in the I.T. training industry, Ricky’s extensive experience comes from teaching as a polytechnic lecturer and corporate instructor.
```
Comment (wfxronald) [inline pages/cyberia.md:47]: I'm just being particular but maybe let's spell it as "IT" instead it "I.T."

File: pages/cyberia.md [hunk 3]
```diff
@@ -40,12 +48,17 @@ Although we are a newly-formed company, our team is made up of veterans who have
...
 
-With his diverse experience and broad knowledge, Ricky is an excellent facilitator whose previous participants have greatly benefitted from his insights. Besides understanding the needs of his participants, Ricky's strengths are his clear, focussed delivery and his ability to generate enthusiasm. His affable personality and cheerful style means his lessons are always interesting and engaging.
+At Ngee Ann Polytechnic, Ricky taught programming languages as well as web design and web development. As a corporate instructor, Ricky has been an Adobe Certified Instructor and Adobe Certified Expert since 2006. Due to his numerous Adobe certifications, he has been conferred the highly coveted title of “Adobe Certified Instructor – Creative Suite Master”. Some of his major clients include GlaxoSmithKline, NOL, AIA, SPH, and Seagate. He has also collaborated with People’s Association in organising and conducting I.T. courses for the public in community clubs.
```
Comment (wfxronald) [inline pages/cyberia.md:49]: Replace "I.T" to "IT" too

**Review by wfxronald — APPROVED** [review]
Looks good in general!

File: pages/cyberia.md (hunk 1 above)
Comment (clement0010) [reply pages/cyberia.md]: This will not be included, so ill be removing it.

**Comment by dcshzj** [comment]
You will need to update the image on https://www.nussucommit.com/events/ as well. Also, it might be better to compress the photo first, 5 MB is too large for a website.

**Pull request merged**